appscan-config.xml

```diff
@@ -0,0 +1,7 @@
+<Configuration>
+  <Targets>
+    <Target path=".">
+      <Exclude>test/</Exclude>
```

Comment:
Should we also exclude the "examples" directory?

Reply:
Added!  `examples` was not causing code scan problems, but it makes sense to add it too.